scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt

```diff
@@ -630,6 +673,12 @@ internal class ScenarioServiceImpl(
       if (datasetListUpdated) {
         publishDatasetListChangedEvent(organizationId, workspaceId, scenarioId, scenario)
       }
+
+      azureDataExplorerClient.ingestScenarioValidationStatus(
+          organizationId,
+          workspace.key,
+          scenarioId,
+          scenario.validationStatus?.toString() ?: ScenarioValidationStatus.Unknown.toString())
```

Comment:
Could be interesting to use validationStatus.value instead of validationStatus.toString(). (and Unknown.value)
toString method can be overridden

Reply:
if we've a good reason(complicated string representation such as joining values) to override toString but here we only need a string value